p2p/peer.go

```diff
@@ -388,6 +405,12 @@ func (p *peer) metricsReporter() {
 						entry.pendingRecvBytes = 0
 					}
 				}
+				for chID, entry := range p.pendingMetrics.perChannelCache {
+					p.metrics.MessageAverageSendDelay.
+						With("peer_id", string(p.ID())).
+						With("channel_id", string(chID)).
+						Set(entry.sendDelay.Seconds() / float64(entry.count))
+				}
```

Comment:
```suggestion
				}
				p.pendingMetrics.perChannelCache = make(map[byte]*peerPendingDelayMetrics)
```
As we want a gauge, we need to reset every time we report